handbook/engineering/distribution/goals.md

```diff
@@ -4,11 +4,21 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 ## Medium-term goals (~3-6 Months)
 
+### [Improve deployment pipeline UX](https://github.com/orgs/sourcegraph/projects/96)
+
+- **Owner**: TBD
+- **Status**: Not started. Unknown amount of work.
+- **Outcomes**:
+  - Its simple to identify if a commit has been deployed to an environment
```

Comment:
```suggestion
  - It is simple to identify when a commit has been deployed to an environment
```

I feel like _if_ is a bit too easy to solve (for example, you can already kind of tell from the build version in `https://sourcegraph.com/users/$ME/settings` ), but much more interesting would be knowing _when_ a commit has been deployed (for example, via a comment on the PR, or through Buildkite, or Slack, or etc.) which seems like it could be nice for triage as well (a question that seems to come up often is, "did something deploy at x time?")

Reply:
Yeah, agree, plus, I'd like to see the history of deployments easily.

Reply:
>I feel like if is a bit too easy to solve 

Thanks, yeah that is what I meant, not only when as in "timeline" but rather, I merged, is my commit already in the environment?

>like to see the history of deployments easily.

Lets add that!

Reply:
Unresolving to highlight the conversation